app/assets/javascripts/controllers/map_controller.js

```diff
@@ -21,12 +20,21 @@
         map.renderLayers();
       });
 
-      layersSpec.fetch().done(function() {
-        // This method triggers an event called 'reset'
-        layersSpec.filterByIds(layersActived);
-      });
+      this.setCollection();
 
       this._initMapComponents();
+
+      App.Events.on('params:update', this.setCollection.bind(this));
+    },
+
+    setCollection: function() {
+      var layersActived = [2];
```

Comment:
I don't like to use numbers.... It would be better a slug. But this is for the future